tools/wpt/testfiles.py

```diff
@@ -252,6 +260,14 @@ def affected_by_wdspec(test):
                     break
         return affected
 
+    def affected_by_interfaces(file_contents):
+        if 'idlharness.js' in file_contents:
```

Comment:
If you compare the time taken to compute affected tests before/after this, is there a difference? Won't this look through the contents of all files? If so, can you exit early if `interfaces_changed_names` is empty?

Reply:
Done.